ApiReview/API.Polly.Core/NoDocs/Polly.Simmy/OutcomeChaosPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Polly.Simmy.Outcomes;
+
+namespace Polly.Simmy;
+
+public static class OutcomeChaosPipelineBuilderExtensions
+{
+    public static ResiliencePipelineBuilder AddChaosFault(this ResiliencePipelineBuilder builder, bool enabled, double injectionRate, Exception fault);
+    public static ResiliencePipelineBuilder AddChaosFault(this ResiliencePipelineBuilder builder, bool enabled, double injectionRate, Func<Exception?> faultGenerator);
+    public static ResiliencePipelineBuilder AddChaosFault(this ResiliencePipelineBuilder builder, OutcomeStrategyOptions<Exception> options);
```

Comment:
```suggestion
    public static TBuilder AddChaosFault<TBuilder>(this ResiliencePipelineBuilder builder, FaultStrategyOptions options) where TBuilder: ResiliencePipelieBuilderBase;
```
Using `OutcomeStrategyOptions<Exception>` feels odd as per this comment:
https://github.com/App-vNext/Polly/pull/1459#discussion_r1305198892

Reply:
I think you created this PR with an old API file bc I already changed that to use `FaultStrategyOptions`.